logstash-core/src/main/java/org/logstash/plugins/NamespacedMetricImpl.java

```diff
@@ -62,13 +64,13 @@ public CounterMetric counter(final String metric) {
 
     @Override
     public NamespacedMetric namespace(final String... key) {
-        final IRubyObject[] rubyfiedKeys = Stream.of(key)
+        final IRubyObject[] rubyFieldKeys = Stream.of(key)
```

Comment:
"rubyfiedKeys" is intentional and not a typo.

"-fied" is past-tense of [the "-fy" suffix](https://www.merriam-webster.com/dictionary/FY#:~:text=%2Dfy-,verb%20suffix,%3A%20cause%20to%20become), so "rubyfiedKeys" means something along the lines of "[Keys] that have been transformed to have qualities related to _[Ruby]_".